api/legacy/reportback/index.js

```diff
@@ -46,20 +44,13 @@ else {
  */
 router.post('/:campaign', function(request, response) {
   var campaignName = request.params.campaign;
+
   logger.log('verbose', '/reportback/%s request.body:', campaignName, JSON.stringify(request.body));
-  var campaignConfig
-    , phone
-    , requestData
-    , i
-    ;
-  
-  // Get the config either from the override or the campaign value in the URL
-  if (request.query.config_override) {
-    campaignConfig = app.getConfig(app.ConfigName.REPORTBACK, request.query.config_override, 'config_override');
-  }
-  else {
-    campaignConfig = app.getConfig(app.ConfigName.REPORTBACK, campaignName, 'endpoint');
-  }
+  var phone;
+  var requestData;
+  var i;
```

Comment:
Leaving this for now, as we're not concerned with linting any of the `api/legacy` code (and looking to deprecate in v3)